internal/local/dependencies.go

```diff
@@ -38,3 +39,29 @@ func FindAppDependencies(path string) (Dependencies, error) {
 
 	return Dependencies{rootDir, archivePath}, nil
 }
+
+// FindPackageJSON finds the Realm app dependencies as a package.json file
+func FindPackageJSON(path string) (Dependencies, error) {
+	app, appOK, appErr := FindApp(path)
+	if appErr != nil {
+		return Dependencies{}, appErr
+	}
+	if !appOK {
+		return Dependencies{}, nil
+	}
+
+	rootDir := filepath.Join(app.RootDir, NameFunctions)
+
+	packageJSONPath, err := filepath.Abs(filepath.Join(rootDir, namePackageJSON))
+	if err != nil {
+		return Dependencies{}, err
+	}
+
+	if _, err := os.Stat(packageJSONPath); err != nil {
+		if os.IsNotExist(err) {
+			return Dependencies{}, fmt.Errorf("package.json not found at '%s'", rootDir)
+		}
+	}
```

Comment:
we should probably still `return Dependencies{}, err` here, rather than ignore it